client/yarn.lock

```diff
@@ -0,0 +1,10167 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is server is initiated with yarn too? If not I think we should use yarn for server too. @nimish-gupta need your inputs on this.